### PR TITLE
Add IDE

### PR DESCRIPTION
**Pull request by styfle**
I answered a question 😄

### CODE REVIEW

**Comment by zachbharris** [comment]
Mentioning VS Code as an IDE is misleading as it isn't an IDE by default (would require extra plugins/extensions). You could word it more along the lines of
> VS Code is a great choice as a text editor that can obtain IDE features through their plugins/extensions

Just my 2 cents

**Comment by styfle** [comment]
Why is VS Code not an IDE?

**Comment by zachbharris** [comment]
Straight from the Visual Studio Code FAQ:
> Visual Studio Code is a streamlined code editor with support for development operations like debugging, task running and version control. It aims to provide just the tools a developer needs for a quick code-build-debug cycle and leaves more complex workflows to fuller featured IDEs. For more details about the goals of VS Code, see Why VS Code.

To more-so answer your question, VS Code is a code editor first with the capability to become an IDE through extensions and plugins, but leaves more complex IDE features to the full-featured IDEs. 

The question you were answering was asking _What IDE should I use?_ and while VS Code may be a sound recommendation, there should be more elaboration on why VS Code is the "IDE" to use and how it would be configured to work as an IDE. I don't disagree that for Front-End development that VS Code is an amazing choice, as it is something that I use as well, however, the question was focused around IDE and not code/text editor. 

Once again, this was just my 2 cents, and the decision is ultimately up to an admin of the repo.

**Comment by styfle** [comment]
The quote mentions many features that come out of the box.
But just like every IDE, you can install extensions for "more complex workflows to fuller featured IDEs".

Using the term "code editor" is mostly marketing speak to make you feel like VS Code is lightweight compared to the huge, slow, ancient IDEs.

But I assure you, VS Code has way more features that Notepad++.

These come out of the box and I'm really not sure what else you expect from an IDE:

- folder tree
- text editing
- syntax highlighting
- auto-completion (intellisense)
- debugging
- task running
- version control
- terminal
- html/markdown rendering
- extensible 

Is there a specific feature you're thinking of that is missing?

**Comment by PhoebePan** [comment]
There is a [Visual Studio IDE](https://visualstudio.microsoft.com/vs/) more powerful than [Visual Studio Code](https://code.visualstudio.com/).

The IDE:
- navigate to related functions
- develop cross devices app in one tool

**Comment by styfle** [comment]
You can do both of those things with VS Code.

**Comment by dan-harris** [comment]
For those arguing that vscode isn't an IDE... fair enough, each to their own 🤷‍♀️. I think IDE's are heavily influenced by your workflow/language/framework. As an example, Visual Studio (as mentioned) really excels at any C#, F# .etc workflow, whereas (_IMHO_) vscode definitely excels within a JS/TS type workflow.

Since there'll never be pure agreement as to what IDE to use, maybe those that have differing opinions do a PR and add their IDE of choice as options?

**Comment by emmabostian** [comment]
I understand that VS Code isn't an IDE but it's still a useful code editor for beginners. Thus, I am changing the wording of the question to include this. Thanks!

File: README.md [hunk 1]
```diff
@@ -38,7 +38,10 @@ Front-End FAQ will be the one place you can go to for all your Front-end questio
  
  # General
  **What IDE should I use?**
```
Comment (emmabostian) [inline README.md]:
```suggestion
 **What IDE or code editor should I use?**   
```

File: README.md [hunk 2]
```diff
@@ -38,7 +38,10 @@ Front-End FAQ will be the one place you can go to for all your Front-end questio
  
  # General
  **What IDE should I use?**   
- _Waiting for response_   
+
+[VS Code](https://code.visualstudio.com) is a great choice and provides a lot of web friendly features out of the box, along with many 3rd party extensions to customize it just the way you like.
+
+If you avoid installing software, try one of these [awesome online IDEs](https://ide.ceriously.com).
```
Comment (emmabostian) [inline README.md:45]: Could you additionally list Sublime, WebStorm, and Atom as alternative? Thanks!

**Review by emmabostian — CHANGES_REQUESTED** [review]
Just a couple minor comments!

**Comment by Dan503** [comment]
This repo is aimed toward junior front end developers and VS Code is the best code editing software there is for front end developers so VS Code is the best answer to this question.

**Comment by emmabostian** [comment]
I agree that VSCode is the best, however we should provide other alternatives.

**Comment by scottquested** [comment]
Would really like to see JetBrains mention in here. These a fully fledged IDE’s. Should be top of the list for and front-end/backend Dev

**Pull request closed without merging**